images/kubekins-e2e/OWNERS

```diff
@@ -5,7 +5,6 @@ reviewers:
 - BenTheElder
 - cpanato
 - jeremyrickard
-- justaugustus
```

Comment:
hmm we generally prefer to have approvers be a subset of reviewers, to spread the review load 🥕

Reply:
+1

Reply:
Fixed.

Reply:
> hmm we generally prefer to have approvers be a subset of reviewers, to spread the review load 🥕

But... if we feel this way as a project, we should probably update https://go.k8s.io/owners#maintaining-owners-files:

> Good examples of OWNERS usage:
>
> - team aliases are used that correspond to sigs
> - there are more `reviewers` than `approvers`
> - the `approvers` are not in the `reviewers` section
> - OWNERS files that are regularly updated (at least once per release)

Reply:
Thanks for fixing.

At the very least this repo / sig testing operates this way. I would encourage all repos to do so, unclear if the org as a whole will do this and I'm a bit weary of trying to effect org-wide policy at the moment.